src/app/static/src/app/components/plots/MapAdjustScale.vue

```diff
@@ -24,21 +24,23 @@ import {ColourScaleType} from "../../store/colourScales/colourScales";
                 <form novalidate>
                     <div class="row p-0 mb-2">
                         <label for="custom-min-input" class="col col-form-label col-2"><span v-translate="'min'"></span></label>
-                        <div class="col pt-1">
+                        <div class="col pt-1" :style="{paddingRight: 5 + 'px'}">
                             <input id="custom-min-input" type="number" :step="step"
                                    v-model.number="colourScaleToAdjust.customMin"
                                    :max="colourScaleToAdjust.customMax"
                                    @change="update" @keyup="update" :disabled="disableCustom">
                         </div>
+                        <p v-if="colourScaleToAdjust.customMin" class="col col-form-label" :style="{paddingLeft: 0}">{{ scaleText}}</p>
```

Comment:
I had a lot of issues trying to get the user's input to accurately reflect what was being shown when the values had been scaled. In the end I just add this info next to the input to show how much their entered value is being scaled by.